crds/jwst/tpns/calpars_all.tpn

```diff
@@ -0,0 +1,10 @@
+# Template file used by certify to check reference files
+# Some fields may be abbreviated to their first character:
+#
+# keytype = (Header|Group|Column)
+# datatype = (Integer|Real|Logical|Double|Character)
+# presence = (Optional|Required)
+#
+# NAME  KEYTYPE  DATATYPE   PRESENCE    VALUES
+#----------------------------------------------------------
+META.INSTRUMENT.NAME        H   C   R    CALPARS
```

Comment:
Is it worth adding any of the other common attributes?  Or is the file pretty well covered by the model schema?

Reply:
Yes. In fact, this should probably include each instruments `all` tpns. Except for instrument, all other parameters will be the instrument-specific ones.